.pipelines/templates/scan-images.yaml

```diff
@@ -1,12 +1,22 @@
 steps:
   - script: |
-      export REGISTRY="e2e"
-      export IMAGE_VERSION="test"
-      OUTPUT_TYPE=docker  make container
+      # install trivy
       wget https://github.com/aquasecurity/trivy/releases/download/v$(TRIVY_VERSION)/trivy_$(TRIVY_VERSION)_Linux-64bit.tar.gz
       tar zxvf trivy_$(TRIVY_VERSION)_Linux-64bit.tar.gz
+
+      # scan provider image
+      export REGISTRY="e2e"
+      export IMAGE_VERSION="test"
+      OUTPUT_TYPE=docker  make container    
       # show all vulnerabilities in the logs
       ./trivy "${REGISTRY}/provider-azure:${IMAGE_VERSION}"
-      
       ./trivy --exit-code 1 --ignore-unfixed --severity MEDIUM,HIGH,CRITICAL "${REGISTRY}/provider-azure:${IMAGE_VERSION}" || exit 1
+
+      # scan arc conformance test image
+      export REGISTRY="e2e"
+      export IMAGE_VERSION="conformance-test"
+      OUTPUT_TYPE=docker  make build-e2e-test arc-conformance-container    
+      # show all vulnerabilities in the logs
+      ./trivy "${REGISTRY}/provider-azure-arc-conformance:${IMAGE_VERSION}-linux-amd64"
```

Comment:
```suggestion
      ./trivy image "${REGISTRY}/provider-azure-arc-conformance:${IMAGE_VERSION}-linux-amd64"
```
can you update the other ones too?
https://github.com/aquasecurity/trivy/discussions/1515

Reply:
done